medusa/clients/nzb/nzbget.py

```diff
@@ -1,64 +1,50 @@
 # coding=utf-8
-# Author: Nic Wolfe <nic@wolfeden.ca>
-
-#
-# This file is part of Medusa.
-#
-# Medusa is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# Medusa is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with Medusa. If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import unicode_literals
 
 import datetime
+import logging
 from base64 import standard_b64encode
 
-from medusa import (
-    app,
-    logger,
-)
+from medusa import app
 from medusa.common import Quality
+from medusa.logger.adapters.style import BraceAdapter
 from medusa.helper.common import try_int
 
 from six.moves.http_client import socket
 from six.moves.xmlrpc_client import ProtocolError, ServerProxy
 
 
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+log = BraceAdapter(log)
```

Comment:
@ratoaq2 I would likely implement it as a classmethod of `BraceAdapter` so you could do `BraceAdapter.getLogger(__name__)` but was more concerned about functionality before convenience methods.